src/models/slices/directanswer.ts

```diff
@@ -1,5 +1,11 @@
 import { FeaturedSnippetDirectAnswer, FieldValueDirectAnswer } from '@yext/answers-core';
 
+/**
```

Comment:
This often difficult to do, but we strive to write documentation that provides more information than the name itself. For example, `DirectAnswerState` could be documented as: 'Maintains the direct answer associated with the latest search'

Reply:
Updated this one and some others to be more informative